content/articles/an-overview-of-web-3-0/index.md

```diff
@@ -0,0 +1,105 @@
+The World wide web is a crucial element of our modern life. It has become part of us, and it would not be easy to outgrow our lives without it. This is because technology has greatly advanced as people can efficiently work remotely and easily communicate with the help of the internet. The reliance on the world wide web has grown in the last three decades and has gone through stages of evolution from [Web 1.O](https://websitebuilders.com/how-to/glossary/web1/), [Web 2.O](https://en.wikipedia.org/wiki/Web_2.0), and [Web 3.O](https://web3.foundation/).
+
+Most people are probably wondering what the future web will be like. Well, Web 3.O (Web 3) is the future web. Many people are confused about what the next generation internet (Web 3) will be like. Web 3 is a part of a much larger phase of transition that we will go through. Therefore, there is a more significant change going on right now on the internet, and lots of improvements are being deployed.
```

Comment:
Using preposition at the end of your sentence? It's less formal though? could you check it out?